src/main/java/edu/oakland/production/database/DatabaseGisManagerStub.java

```diff
@@ -0,0 +1,23 @@
+package edu.oakland.production.database;
```

Comment:
Can we go through and add a private variable and a getter to the stub to store the mode, so we don't need to change the public interface, but can still test what that the request works?